ResearchKit/ActiveTasks/ORKAudioContentView.m

```diff
@@ -47,7 +47,7 @@ @interface ORKAudioGraphView : UIView
 @property (nonatomic, strong) UIColor *keyColor;
 @property (nonatomic, strong) UIColor *alertColor;
 
-@property (nonatomic, copy) NSArray *values;
+@property (nonatomic, strong) NSArray *values;
```

Comment:
You're correct. When I made the changes based on this comments, I didn't realize that one `NSArray` property was already intentionally set to `copy` on your end.

Submitted a change to set it back to `copy` :)